src/sphinx_book_theme/header_buttons/launch.py

```diff
@@ -189,6 +190,22 @@ def add_launch_buttons(
                 }
             )
 
+    if lightning_studio_url:
+        if provider.lower() != "github":
+            SPHINX_LOGGER.warning(f"Provider {provider} not supported on Lightning.")
+        else:
+            github_path = f"%2F{org}%2F{repo}%2Fblob%2F{branch}%2F{path_rel_repo}"
```

Comment:
I think we probably need to URL escape these.

Reply:
@agoose77 is the latest commit what you were looking for? At least the tests seem to pass again. Let me know if I need to change something else.

Reply:
Hi @nilsleh not quite -- I'm worried about the case where `org`, `repo`, `branch`, or `path_rel_repo` aren't URL escaped properly. 

Equally, it's something that the other launchers would need to worry about, so I'll just merge for now!